### PR TITLE
Update user definitions page

### PR DESCRIPTION
**Pull request by ebrodymoore**

### CODE REVIEW

File: content/bizops/user_definitions.md [hunk 1]
```diff
@@ -1,54 +1,60 @@
...
+| Deleted                        | Users whose account was deleted this month                                                        | ✔️    | ✔️      | `growth_statistics.DeletedUsers`        |
 
-There is also a special state, **activated**, which is permanent once triggered and can apply to any of the above states. For example, you can have an activated active user or even an activated deleted user.
+### Visitor
 
-### What is a visitor?
+A visitor is a unique person who showed up on the site and did anything (or nothing at all, apart from viewing the page). Currently a unique person cannot be identified between our on-premises & cloud solution; in other words, when aggregating data across our deployment solution one person could be counted as a "unique person" twice. It is important to measure unique visitors because each of these visitors is potentially an active user. It is equally important that we don't count anyone who visits the product as an active user so that we can measure success of our features that are intended to convert them into an active user.
```
Comment (rrhyne) [inline content/bizops/user_definitions.md]:
```suggestion
A visitor is a unique person who visited the site. There is no requirement for the visitor to perform an action, apart from viewing a single page. Currently, a unique person cannot be identified between our on-premises & cloud solution; in other words, when aggregating data across our deployment solution one person could be counted as a "unique person" twice. It is important to measure unique visitors because each of these visitors is potentially an active user. It is equally important that we do not count anyone who visits the product as an active user so that we can measure success of our features that are intended to convert them into an active user.
```

File: content/bizops/user_definitions.md [hunk 2]
```diff
@@ -1,54 +1,60 @@
...
-| Created     | Users whose account was created this month                       | ✔️    | ✔️      |
-| Deleted     | Users whose account was deleted this month                       | ✔️    | ✔️      |
+| Metric                         | Description                                                                                       | Cloud | On-prem | Pings data point                        |
+| ------------------------------ | ------------------------------------------------------------------------------------------------- | ----- | ------- | --------------------------------------- |
+| [Visitor](#visitor)            | Anyone who accessed the product                                                                   | ✔️    | ✔️      | `site_activity.MAU.UserCount`           |
+| [Active](#active-user)         | Users who performed a qualifying activation event (see below)                                     | ✔️    | n/a     | n/a                                     |
+| [Activated](#activated-user)   | Users who have completed a specific series of actions that indicate they have developed the habit | ✔️    | n/a     | n/a                                     |
```
Comment (jasonyavorsky) [inline content/bizops/user_definitions.md]:
```suggestion
| [Activated](#activated-user)   | Users who have completed a specific series of actions that indicate they have developed the habit. This is a permanent state once applied. | ✔️    | n/a     | n/a                                     |
```

File: content/bizops/user_definitions.md [hunk 3]
```diff
@@ -1,54 +1,60 @@
...
+We use metrics to guide prioritization and planning. By defining metrics against the kinds of behaviors that we want to drive, we can plan features and measure results against the outcomes we're looking for. The three most important metrics we track are acquisition (of visitors), activation (visitor experiences habit creation), and retention (user remains active).
 
 ## User states
 
-People using Sourcegraph can be segmented into a number of different states, the following of which are relevant over a measured time period (for example, monthly or weekly):
+People using Sourcegraph can be segmented into a number of different states, the following of which are relevant over a measured time period (for example, monthly or weekly). They are not mutually exclusive in some cases - a user can be both registered, activated and churned at the same time.
```
Comment (jasonyavorsky) [inline content/bizops/user_definitions.md:7]: Maybe add a column indicating which ones can be are mutually exclusive and which are not? It's one of the most confusing things that this is unclear/inconsistent.

Sentences like "We have an activated inactive churned then deleted former visitor turned user" are hard to parse otherwise.

**Comment by jasonyavorsky** [comment]
Nice improvements, great job!

File: content/bizops/user_definitions.md (hunk 1 above)
Comment (jasonyavorsky) [inline content/bizops/user_definitions.md]:
```suggestion
A visitor is a unique person who showed up on the site and did anything, even just viewing the page. Currently a unique person cannot be identified between our on-premises & cloud solution; in other words, when aggregating data across our deployment solution one person could be counted as a "unique person" twice. It is important to measure unique visitors because each of these visitors is potentially an active user. It is equally important that we don't count anyone who visits the product as an active user so that we can measure success of our features that are intended to convert them into an active user.
```

**Review by jasonyavorsky — APPROVED** [review]
Approved by me, for what it's worth :)

**Review by rrhyne — APPROVED** [review]
Nice updates.

File: content/bizops/user_definitions.md (hunk 3 above)
Comment (ebrodymoore) [reply content/bizops/user_definitions.md:7]: Good callout. Will come up with a proposed solution

Comment (ebrodymoore) [reply content/bizops/user_definitions.md:7]: Added this section in the latest commit. Let me know your thoughts!
